services/sentry/eventAggregator.js

```diff
@@ -64,6 +72,14 @@ function makeRecorder(channelId) {
 			return hasAccess
 		}
 
+		const priceRuleModifyEvs = events.filter(x => x.type === eventTypes.update_price_rules)
+		if (priceRuleModifyEvs.length) {
+			// if there are multiple evs only apply the latest
+			await updateChannelPriceMultiplicationRules(priceRuleModifyEvs[priceRuleModifyEvs.length - 1])
```

Comment:
I think it's much better to not use a lambda function here but to call `await channelsCol.updateOne(` directly..it would be much more readable